prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/PMMLTest.scala

```diff
@@ -0,0 +1,31 @@
+package pl.touk.nussknacker.prinz.pmml
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.prinz.pmml.repository.PMMLModelRepository
+
+import java.io.{BufferedReader, InputStreamReader}
+
+// TODO to be removed after full repository implementation
+object PMMLTest extends App with LazyLogging {
+  val d1  = "file:///home/avan1235/Desktop/pmml/fraud-detection.pmml"
+  val d2  = "file:///home/avan1235/Desktop/pmml/"
+  val d3  = "http://localhost:5100"
+  val d4  = "http://localhost:5100/fraud-detection.pmml"
+  implicit val config: Config = ConfigFactory.parseString(s"""
+      |  pmml {
+      |    modelsDirectory: "$d4"
+      |    modelDirectoryHrefSelector: "body > ul > li > a"
```

Comment:
Third time here. I think JSON would work better.

Reply:
as in previous comment - leave it for now to have working repository for pmml and create an issue with some idea how to solve this with json